.github/workflows/misc-migtests.yml

```diff
@@ -59,6 +59,7 @@ jobs:
         run: |
           cd installer_scripts
           yes | ./install-yb-voyager --install-from-local-source --only-pg-support
+          echo "/usr/lib/postgresql/16/bin" >> "$GITHUB_PATH"
```

Comment:
Oh okay, Github actions have there own `GITHUB_PATH` to update. cc: @ShivanshGahlot